airbyte-integrations/connectors/source-salesforce/source_salesforce/api.py

```diff
@@ -0,0 +1,348 @@
+    "ContentFolderMember",
+    "ContentFolderItem",
+    "SearchLayout",
+    "SiteDetail",
+    "EntityParticle",
+    "OwnerChangeOptionInfo",
+    "DataStatistics",
+    "UserFieldAccess",
+    "PicklistValueInfo",
+    "RelationshipDomain",
+    "FlexQueueItem",
+    "NetworkUserHistoryRecent",
+    "FieldHistoryArchive",
+    "ColorDefinition",
+    "AppTabMember",
+    "FlowVersionView",
+    "IconDefinition",
+    "FlowVariableView",
+    "DatacloudDandBCompany",
+]
+
+# The following objects are not supported by the query method being used.
+QUERY_INCOMPATIBLE_SALESFORCE_OBJECTS = [
+    "DataType",
+    "ListViewChartInstance",
+    "FeedLike",
+    "OutgoingEmail",
+    "OutgoingEmailRelation",
+    "FeedSignal",
+    "ActivityHistory",
+    "EmailStatus",
+    "UserRecordAccess",
+    "Name",
+    "AggregateResult",
+    "OpenActivity",
+    "ProcessInstanceHistory",
+    "OwnedContentDocument",
+    "FolderedContentDocument",
+    "FeedTrackedChange",
+    "CombinedAttachment",
+    "AttachedContentDocument",
+    "ContentBody",
+    "NoteAndAttachment",
+    "LookedUpFromActivity",
+    "AttachedContentNote",
+    "QuoteTemplateRichTextData",
+    "DatacloudAddress",
+    "OrgLifecycleNotification",
+    "AssetTokenEvent",
+    "RemoteKeyCalloutEvent",
+    "CredentialStuffingEvent",
+    "BatchApexErrorEvent",
+    "LoginEventStream",
+    "ReportEventStream",
+    "AIPredictionEvent",
+    "SessionHijackingEvent",
+    "PlatformStatusAlertEvent",
+    "ApiEventStream",
+    "LightningUriEventStream",
+    "LogoutEventStream",
+    "FlowExecutionErrorEvent",
+    "AsyncOperationEvent",
+    "BulkApiResultEvent",
+    "LoginAsEventStream",
+    "ConcurLongRunApexErrEvent",
+    "ApiAnomalyEvent",
+    "UriEventStream",
+    "ProcessExceptionEvent",
+    "ListViewEventStream",
+    "ReportAnomalyEvent",
+    "AsyncOperationStatus",
+]
+
+
+UNSUPPORTED_BULK_API_SALESFORCE_OBJECTS = [
+    "Attachment",
+    "CaseStatus",
+    "ContractStatus",
+    "DeclinedEventRelation",
+    "FieldSecurityClassification",
+    "OrderStatus",
+    "PartnerRole",
+    "RecentlyViewed",
+    "ServiceAppointmentStatus",
+    "SolutionStatus",
+    "TaskPriority",
+    "TaskStatus",
+    "UndecidedEventRelation",
+    "AcceptedEventRelation",
+    "AssetTokenEvent",
+    "AttachedContentNote",
+    "EventWhoRelation",
+    "QuoteTemplateRichTextData",
+    "TaskWhoRelation",
+]
+
+
+UNSUPPORTED_FILTERING_STREAMS = [
+    "ApiEvent",
+    "BulkApiResultEventStore",
+    "EmbeddedServiceDetail",
+    "EmbeddedServiceLabel",
+    "FormulaFunction",
+    "FormulaFunctionAllowedType",
+    "FormulaFunctionCategory",
+    "IdentityProviderEventStore",
+    "IdentityVerificationEvent",
+    "LightningUriEvent",
+    "ListViewEvent",
+    "LoginAsEvent",
+    "LoginEvent",
+    "LogoutEvent",
+    "Publisher",
+    "RecordActionHistory",
+    "ReportEvent",
+    "TabDefinition",
+    "UriEvent",
+]
+
+
+class Salesforce:
+    version = "v52.0"
```

Comment:
this version appears twice in two different files, can we centralise this to avoid confusion?